### DISCUSSION TITLE
How to run the benchmarker with Spring Boot?

### DISCUSSION THREAD

**Discussion by PatrickDiallo23** [TimefoldAI/timefold-solver#645] [General]
The documentation lacks an example on how to run the benchmarker with Spring Boot using timefold-solver-benchmark module. Can you provide an example, please?

**Comment by triceo** [comment]
Hello @PatrickDiallo23. 

Much like the `Solver`, you can inject the `PlannerBenchmarkFactory` and build your benchmark from that. There are also some benchmark-specific [configuration properties](https://timefold.ai/docs/timefold-solver/latest/integration/integration#integrationWithSpringBootProperties) you may be interested in.
